### PR TITLE
Update ugdg

### PR DESCRIPTION
**Pull request by Yen-Peng**
- [x] update model class diagram image

- [x] update ugdg for sorting feature

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -366,7 +366,7 @@ Deletes a person with the NRIC S98654322D in ORGANice.
 === Detecting matches: `match`
 
 This command runs a kidney matching test on patients and donors in ORGANice and displays the match results.
-Refer to <<FAQ>> for a better understanding of the procedure for kidney matching.
+Refer to <<Glossary>> for a better understanding of the procedure for kidney matching.
```
Comment (WilliamRyank) [inline docs/UserGuide.adoc]: FAQ seems correct here why change to `Glossary`

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -562,6 +583,9 @@ that contains the data of your previous ORGANice folder.
 2. Do a tissue typing test to determine the HLA antigens of the patient and the donor.
 3. Do a cross match, which is a trial transfusion.
 
+*Q*: What is the success rate and how is it computed? +
+*A*:
```
Comment (C-likethis123) [inline docs/UserGuide.adoc]: The success rate measures the tissue compatibility between a donor and a patient. It is computed via this formula: (number of common tissue types between donor and patient) / 6 * 100%

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -400,29 +399,51 @@ To match all the patients in ORGANice, do the following:
...
 +
 
+Following this, you can sort the matches based on different categories to deduce which patient to prioritize.
++
+Refer to refer to <<Sorting matches: `sort`>> for more information.
 
 === Sorting matches: `sort`
-Users can specify how they want the patient-donor pairs to be sorted. +
-Sorting is done by passing in optional parameters to the `match` command. +
+This section specifies how a list of matches can be sorted to determine which patient to prioritize for organ transplant.
```
Comment (WilliamRyank) [inline docs/UserGuide.adoc]: Better tone to sound as if you are talking with the user? Such as, `After getting the list of potential donors, you can ........`

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -400,29 +399,51 @@ To match all the patients in ORGANice, do the following:
...
 
-Matches can be sorted according to three categories:
+**The sort command can only be used after a match command** to sort the resulting list of patient-donor pairs generated.
+The following specifies three categories by which you can sort the matches.
```
Comment (WilliamRyank) [inline docs/UserGuide.adoc]: I expect 3 bullet points if you said this, so can add in the 3 categories as bullet points her maybe

File: docs/UserGuide.adoc (hunk 1 above)
Comment (Yen-Peng) [reply docs/UserGuide.adoc]: Amended.

File: docs/UserGuide.adoc (hunk 2 above)
Comment (Yen-Peng) [reply docs/UserGuide.adoc]: Added.

File: docs/UserGuide.adoc (hunk 3 above)
Comment (Yen-Peng) [reply docs/UserGuide.adoc]: Amended.

File: docs/UserGuide.adoc (hunk 4 above)
Comment (Yen-Peng) [reply docs/UserGuide.adoc]: Added.

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -400,29 +399,59 @@ To match all the patients in ORGANice, do the following:
...
+After matching patients and donors, you can sort the list of matches to determine which patient to prioritize for organ transplant.
 
-Matches can be sorted according to three categories:
+**The sort command can only be used after a match command** to sort the resulting list of patient-donor pairs generated.
+There are three categories by which you can sort the matches, namely:
 
-1. Success rate of match
-2. Priority of patient
-3. Date of expiry of donors' organs
+1. Sorting based on donor's organ expiry date
```
Comment (WilliamRyank) [inline docs/UserGuide.adoc]: i prefer bullet points though cause order here doesnt matter. What do you think?

Comment (Yen-Peng) [reply docs/UserGuide.adoc]: changed.

**Pull request merged**